assets/settings/default.json

```diff
@@ -681,4 +693,4 @@
   //   - `long`: "2 selections, 15 lines, 32 characters"
   // Default: long
   "line_indicator_format": "long"
-}
+}
```

Comment:
Let's keep the trailing newline at least.
Those `language_servers` formats happened for me too, so might be ok to keep.

Reply:
I've kept the language_server formats and kept the newline.